src/mesos_state_client.erl

```diff
@@ -51,13 +102,7 @@ poll(URI) ->
   {ok, Hostname} = inet:gethostname(),
   UserAgent = lists:flatten(io_lib:format("Mesos-State / Host: ~s, Pid: ~s", [Hostname, os:getpid()])),
   Headers = [{"Accept", "application/json"}, {"User-Agent", UserAgent}],
-  Headers1 =
-    case application:get_env(?APP, token) of
-      undefined ->
-        Headers;
-      {ok, Value} ->
-        [{"Authorization", Value}|Headers]
-    end,
+  Headers1 = maybe_use_token(Headers),
```

Comment:
In the future tokens will be rotated more frequently, what's the benefit of doing it that way and then changing it back to something like this later?